examples/hypb-example.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: "kubeflow.org/v1alpha1"
+kind: StudyController
```

Comment:
Could we name it Study?
I think studycontroller will confuse users. Our controller is also study controller.

Reply:
I want to distinguish between resource and process.
The Study and Trial are resources that should be saved to DB persistently.
The Worker and StudyController(now) are a process of Trial and Study each other.
Since the process is ephemeral, I want to make them CRD.

I agree StudyController is confusing. We should rename it.
Do you have any idea for the name?
For example, Experiments, StudyJob.

Reply:
I renamed StudyController to StudyJob and StudyJob-Controller.